src/Symfony/Component/HttpClient/composer.json

```diff
@@ -37,7 +37,9 @@
         "php-http/httplug": "^1.0|^2.0",
         "psr/http-client": "^1.0",
         "symfony/amphp-http-client-meta": "^1.0|^2.0",
+        "symfony/cache": "^6.4|^7.0",
         "symfony/dependency-injection": "^6.4|^7.0",
+        "symfony/filesystem": "^6.4|^7.0",
```

Comment:
I don't see any new direct dependency on the filesystem component in this code. Is this an indirect dependency for the cache in tests?

Reply:
It was. I thought I removed it. Adressed in https://github.com/symfony/symfony/pull/59576/commits/18408cfcb50c6568cf693168711fb31772ce6502